pkg/packagekit/updates.jsx

```diff
@@ -815,6 +990,20 @@ class OsUpdates extends React.Component {
                 onAction={ () => cockpit.jump("/subscriptions", cockpit.transport.host) } />;
         }
 
+        const tracerWarning = () => {
+            if (this.state.rebootRequired)
+                return <Alert variant="warning" isInline title={_("System needs reboot.")} />;
+            else if (this.state.restartableServices.systemctl || this.state.restartableServices.manual)
+                return <Alert variant="warning"
+                    isInline
+                    title={_("Some services need to be restarted.")}
```

Comment:
They need to be restarted not for their own sake, but so that the updates can take effect.
I think the string should be: `Some services need to be restarted for the updates to take effect`